ironfish/src/network/peers/connections/webSocketConnection.ts

```diff
@@ -102,6 +102,8 @@ export class WebSocketConnection extends Connection {
         return
       }
 
+      this.metrics?.p2p_InboundTrafficByMessage.get(message.type)?.add(byteCount)
```

Comment:
any reason not to put these with the others so all the same code is together?

Reply:
Nope I'll combine. Also was wondering if we should record `p2p_InboundTraffic` even if the message is not parsed correctly. Don't need to change it now but something to consider